### DISCUSSION TITLE
Network error 99 on Windows client for a specific folder

### DISCUSSION THREAD

**Discussion by janaxhell** [nextcloud/all-in-one#2305] [Questions] [answered]
I am getting network error 99 on hundreds of TXT files from a specific folder. Everything else in other folders, TXT files included, give no problem, these ones make a mess.
I have Nextcloud AIO 25.0.4 on a x64 OMV-Docker server and clients on both Android and Windows. I do not share that folder with Android, only between OMV and Windows.
On this page [https://help.nextcloud.com/t/syncing-client-crashes-with-network-error-99/149047/22](https://help.nextcloud.com/t/syncing-client-crashes-with-network-error-99/149047/22) a guy says he's solved by 
- On the client, disable any upload speed limits _[I have no limit set on my Windows client]_
- On the server, edit config.php to add the line 'bulkupload.enabled' => false

but on Nextcloud AIO there are many containers and config.php's, which one should I use? 
There is absolutely nothing wrong with those files, they are plain TXT on a perfectly healthy SSD, with no strange letters in filename.

EDIT
I have closed Windows client, uploaded all problematic files via webDAV (Total Commander+WebDav plugin) with NO problem, restarted client. All files were checked as "modified since last check", but in the end green-tagged as synced. This does not solve the problem, it's just a workaround. 

EDIT 2
No, after some time I checked client again and it was claiming that all those already uploaded files cannot be synced and they are "in conflict". It created duplicates with a random alphanumeric string extension.

**Comment by szaimen** [comment, marked as answer]
> * On the server, edit config.php to add the line 'bulkupload.enabled' => false
> 
> but on Nextcloud AIO there are many containers and config.php's, which one should I use?

See https://github.com/nextcloud/all-in-one#how-to-edit-nextclouds-configphp-file-with-a-texteditor or https://github.com/nextcloud/all-in-one#how-to-run-occ-commands, so e.g. `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:system:set bulkupload.enabled --value=false --type=bool` should work.

Apart from that sounds this like an issue that should either be addressed by the [desktop client](https://github.com/nextcloud/desktop/issues) or the [Nextcloud server](https://github.com/nextcloud/server/issues)

**Comment by janaxhell** [comment]
Thanks, the command solved the issue, but I see it's addressed in desktop client as well with basically same trick.
Anyway, although everything in that folder was uploaded, the folder itself is shown in the client as "No Access" (Accesso negato) after having uploaded its contents.

It's the "- Brain Files" folder: all its contents were synced, then that message appeared. The folder has no particular permission, it's on WIndows inside /Documents, I can rwx from anywhere. And I can rwx its contents on server counterpart too, so it seems a bogus warning.

![nextcloud client](https://user-images.githubusercontent.com/55061479/230760015-b5a3aaf5-ea46-49a2-9e75-d1264ef2936e.jpg)

The weird thing is that while all other synced folders on Windows have the green tick on the icon, "- Brain Files" hasn't. It's synced and working, yet NC says No Access and no green tick.

![TOTALCMD64_DMoQdz2b0L](https://user-images.githubusercontent.com/55061479/230760761-dc04382d-08e2-40bc-9087-bdb16ad6d2d3.png)

**Reply by szaimen** [reply]
As I said, this needs to get fixed in the desktop client then. Feel free to open a new issue there if there shouldnt be one already. See https://github.com/nextcloud/desktop/issues
